snap/snapcraft.yaml

```
@@ -1,7 +1,27 @@
 name: authd-google
-summary: Google Broker for authd
+summary: Authd - Google IAM Cloud Authentication for Ubuntu
 description: |
-  Broker that enables OIDC authentication with Google for authd.
+  Authd is a powerful authentication service for Ubuntu, designed to integrate with cloud identity providers like
+  Google IAM. It delivers a secure, flexible solution for organizations and individuals who are transitioning
+  to cloud-based identity management on Ubuntu workstations and servers.
+  Authd uses the Oauth Device Authorization Grant and ensures a consistent and secure login experience across
```

Nitpick: It's spelled "OAuth" not "Oauth". I see that the description in the snap store also uses Oauth, so I think we should also change it there.

```suggestion
  Authd uses the OAuth Device Authorization Grant and ensures a consistent and secure login experience across
```

---

Thanks for spotting it! Normally, once we merge this and I enable "override from snapcraft file", the store description should be synced too (but I will do that after manual  testing).